blackjax/kernels.py

```diff
@@ -757,7 +763,7 @@ def one_step(carry, xs):
 
         return (
             (new_state, new_adaptation_state),
-            (new_state, info, new_adaptation_state),
+            AdaptationInfo(new_state, info, new_adaptation_state),
```

Comment:
Here (and below) we are returning 2 tuple/namedtuple with similar structure - should we just return `AdaptationInfo` instead and do unpacking below?

Reply:
What are you suggesting here exactly? I'm not sure I understand.

Reply:
I am suggesting to return 
```
        return AdaptationInfo(new_state, info, new_adaptation_state)
```

Reply:
I like to only carry-over the values that are necessary in the `scan` loop. This often leads to repetitions but hasn't bothered me so far. What bothers you here?

Reply:
oh right that's for using in `scan` - yeah than it is almost unavoidable